src/main/kotlin/com/ajaxproject/warehouse/dto/WaybillDataDto.kt

```diff
@@ -1,13 +1,18 @@
 package com.ajaxproject.warehouse.dto
 
-import com.ajaxproject.warehouse.entity.Customer
-import com.ajaxproject.warehouse.entity.WaybillProduct
 import java.time.LocalDate
 
 data class WaybillDataDto(
-    val id: Int?,
-    val customer: Customer,
-    val date: LocalDate?,
-    val productList: List<WaybillProduct>,
-    val totalPrice: Double
-)
+    val id: String,
+    val customer: CustomerDataLiteDto,
+    val date: LocalDate,
+    val totalPrice: Double,
+    val products: List<WaybillProductDataDto>
+) {
```

Comment:
why this DTO doesn't have validations like another DTOs have?

Reply:
Because this DTO is for response, it comes from server side, so we don't need to validate it, like we do with request DTOs